### ISSUE TITLE
[bug]: Repeated error message in notification

### ISSUE THREAD

**Issue by bbqchef** [Start9Labs/embassy-os#971]
### Prerequisites

- [X] I have searched for [existing issues](https://github.com/start9labs/embassy-os/issues) that already report this problem, without success.

### EmbassyOS Version

0.3.0

### Device

Laptop/Desktop

### Device OS

Windows

### Device OS Version

Windows 10

### Browser

Firefox

### Browser Version

95.0

### Current Behavior

![EOS Cant clear notification](https://user-images.githubusercontent.com/96219817/146490756-13aab2e7-5efb-4dab-81cf-a33975700461.PNG)

### Expected Behavior

Either
1) Notification text only shows once
2) Notification manages to inform user the error occurred multiple times.

### Steps to Reproduce

I don't know what caused bitcoin core to crash or how to trigger notifications on demand.

**Comment by chrisguida** [comment]
I'm not sure I understand. Is the problem that you want to receive a notification every time bitcoind crashes? We've debounced notifications precisely so that the user doesn't get spammed with notifications if, for instance, bitcoind is an a crash loop.

**Comment by bbqchef** [comment]
Agreed that raising a notification at every crash would risk overcrowding the notifications. Seeing the same sentence repeated in the notification, I thought it was a bug. From your comment I gather this is a design choice, so not a bug.

**Comment by chrisguida** [comment]
Are you ok with this as a design choice? We don't want to confuse people.

**Comment by dr-bonez** [comment]
@chrisguida the service crash notification only contains stderr from a single run of the application. The error message shown contains repeat error messages coming from a single instance of bitcoin. It may be worth looking into being a little more careful about what gets logged to stderr.

**Comment by ProofOfKeags** [comment]
This ticket needs to be moved to the Start9Labs/bitcoind-wrapper repo
